archive/c/c-plus-plus/README.md

```diff
@@ -9,6 +9,7 @@ Welcome to Sample Programs in C++!
 - [Reverse String in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/419)
 - [Fibonacci Sequence in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/496)
 - [Bubble Sort in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/1135)
+- [prime_number in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/1227)
```

Comment:
```suggestion
- [Prime Number in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/1227)
```

Reply:
I changed the code according to the testing